### DISCUSSION TITLE
Can't get any weapon or shield

### DISCUSSION THREAD

**Discussion by henres** [jordanbtucker/totk-no-loot-boxes#10] [Q&A] [answered]
On version 1.2.0 on yuzu 3599, firmware 1.6.2.
I can't get any weapon or shield.
Only the fabrics and armor.

**Comment by jordanbtucker** [comment]
- Which amiibo are you using?
- Have you completed the Great Sky Island and reached Hyrule?
- Have you visited the Depths at least once?

**Reply by henres** [reply]
Which amiibo are you using?
All the one listed with weapon and shield.

Have you visited the Depths at least once?
Nope, didn't know I have to, when I searched online, `Have you completed the Great Sky Island and reached Hyrule?` was mentioned but not the last part.

**Reply by jordanbtucker** [reply, marked as answer]
Yeah, none of the amiibo drop rare weapons, bows, or shields until you visit the Depths for the first time. Since this mod only drops rare items, you have to visit the Depths before you can get any.

However, I just released v2.0.0 of the mod which removes the progression requirements and makes the amiibo more useful after you've obtained the fabric and armor.
